core/server/src/main/java/alluxio/worker/netty/BlockDataServerHandler.java

```diff
@@ -80,14 +79,10 @@ void handleBlockReadRequest(final ChannelHandlerContext ctx, final RPCBlockReadR
     final long lockId = req.getLockId();
     final long sessionId = req.getSessionId();
 
-    BlockReader reader;
+    BlockReader reader = null;
     DataBuffer buffer;
     try {
       reader = mWorker.readBlockRemote(sessionId, blockId, lockId);
```

Comment:
It was introduced in this commit: https://github.com/Alluxio/alluxio/commit/0544462b09b77138a48f4545e7702daeec6967a8

but I don't remember why. Maybe @uronce-cc might know?